_includes/nav.html

```diff
@@ -1,9 +1,10 @@
 <div class="menu-wrapper" id="navigation-menu">
   <nav>
     {% for item in site.data.nav %}
-      <a href="#{{ item.link }}" {% if page.url == item.link %}class="active"{% endif %}>
+      <a href="index.html#{{ item.link }}" {% if page.url == item.link %}class="active"{% endif %}>
         {{ item.name }}
       </a>
     {% endfor %}
+    <a href="coc.html">COC</a>
```

Comment:
Please put all the links in `site.data.nav` whether it be anchor links, relative link or external link used in navigation to keep a single source of data.